File: Ch1-Fundamentals/0009-Multiples-of-Six/main.py
num = 6
while ( num <= 60000 ):
    print(num)
    num += 6

# * Good, but not efficient for this algo's purposes
i = 1
six_count = 0
while (i <= 60000):
    if (i % 6 == 0):
        # print(i)
        six_count += 1
    i += 1
print(six_count)

File: Ch1-Fundamentals/0009-Multiples-of-Six/main.js
let num = 6;
while (num <= 60000) {
    console.log(num);
    num += 6;
}

// * Good, but not efficient for this algo's purposes
let i = 1
let sixCount = 0;
while (i <= 60000) {
    if ( i % 6 === 0 ) {
        // console.log(i);
        sixCount++;
    }
    i++;
}
console.log(sixCount);